1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+<img src="https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png" width="400"  alt="Sonar screen">
+
+## Identify Basic Internet Problems with "ping"
+
+Next time you call your help desk, do you want to wow them with your networking knowledge? Using a command called “ping”, built right into your existing Mac or Windows computer, will help identify basic connection problems. Okay, this might not be enough to “wow” your fellow team members, however they will appreciate that you started the debug process.
```

Comment:
> Next time you call your help desk, do you want to wow them with your networking knowledge?

A great way to start, because it shows wit and captures a reader in. At the same time, help desk people get offended if you do anything other than what they ask you to do.

Seriously, I have tried!

Maybe you can phrase it a bit differently?

Something like
> How about you figure out the problem before you press the right button sequence to get to your IT help desk?